### ISSUE TITLE
Generate word index

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#154]
Wäre schon für die Qualitätssicherung gut zu haben, um so schöne Dinge wie 

> WirrwarrnetzumstrickterQuaalen

(https://github.com/faustedition/xml/issues/149#issue-271210981)
zu finden.

Dann aber auch ein schönes Feature für faust-web.

**Comment by thvitt** [comment]
Fürs Zweite auch faustedition/faust-web#196

**Comment by gerritbruening** [comment]
Daran dachte ich.

**Comment by gerritbruening** [comment]
Siehe http://dev.faustedition.net/word-index

**Comment by gerritbruening** [comment]
Wäre es möglich, aus dem provisorischen Wortindex die Testhandschrift auszuschließen? Sie generiert immerhin 630 tokens.
Wäre es möglich, zunächst tatsächlich die nur einmal vorkommenden Tokens zu listen?

**Comment by thvitt** [comment]
Wenn Du mal mehr willst:
https://github.com/faustedition/faust-gen-html/blob/ee06cc1eb4737d83b9fc1a9715c5122a2b131019/xslt/word-index.xsl#L13

**Comment by gerritbruening** [comment]
Ah, danke!

**Comment by dietmarpravida** [comment]
Sollten wir das nicht auch in die Edition integrieren?

**Comment by gerritbruening** [comment]
Fände ich gut. @thvitt?

**Comment by thvitt** [comment]
Was genau soll da rein? [Das hier](http://dev.faustedition.net/word-index)? Dann überlegt euch halt, wo Ihr’s verlinkt haben wollt, aktuell gehalten wird das

**Comment by thvitt** [comment]
hm, braucht vermutlich noch arbeit
- links zu drucken
- before_, todo in n
Wollt ihr das wirklich? sind halt auch artefakte drin wie `wis` etc.

**Comment by gerritbruening** [comment]
> Was genau soll da rein? [Das hier](http://dev.faustedition.net/word-index)?

Ja, einmal nach Häufigkeit (auf- und absteigend sortierbar), und einmal alphabetisch, fände ich gut. Rückläufig alphabetisch hätte auch was:
* A
* da
* Ha
* Runda 
* ...

**Comment by gerritbruening** [comment]
> artefakte drin wie `wis` etc.

Die Artefakte (= https://github.com/faustedition/faust-web/issues/196?) wäre gut zu kennen, wie auch Quatschwörter wie WirrwarrnetzumstrickterQuaalen.

**Comment by thvitt** [comment]
da sind doch nur tokens drin die einmal vorkommen, da ist die Sortierung nach Häufigkeit trivial. Sonst sind das aber wieder lauter Featurewünsche die wir nicht priorisiert haben …
